README.md

```diff
@@ -1,6 +1,12 @@
-# SpeedRunMod
-hello world
+# TerrariaSpeedrunVerificationMod
```

Comment:
In addition to these, answer some questions someone might commonly ask about the mod. Is there a way to set up when the timer ends? How will the user set when the timer should end? Will this support non-boss runs like all-bosses and/or night's edge? Or will the timer run until you turn it off via keystroke or some other input. Considering that some runs turn off the game to "hard-dupe" would the mod be able to handle this behavior?

Reply:
I think the mod should cover all the catagories that are listed for terraria including nights edge and all bosses. about the timer, regarding the earlier comment. if its a necessary part of the verification then i think it should save all of the objective times and display them in the end or whenever the user wants to check them in a similar fashion to bosschecklist. regardless of when or if the user decides to stop the timer. I don't think it should stop on its own or that the runner should decide before the run what run he'll be doing because somtimes people change the run in the middle or go for a longer run or multiple runs at once. all the while having the timer run in the backround making sure people don't speed the game footage up. If its not mandatory then I think it should be pretty much the same but with an option to turn off.

about the hard dupe. I think that we will be saving the times that bosses die and the start time in some place anyway so it should be pretty simple to check if a start time for a world or charecter exists and if it does to simply keep going from there. But I'm not sure. what do you think?

Reply:
So we'd have a command for the user to turn the timer off when they're done. We will need a small amount of key-binding (doesn't have to been in the 1st version) for this and any other keys they need while running. 

The hard-dupe question is pretty difficult. I know JavaScript uses Epoch millisecond time as a global verifier aka there's a single source we can touch from anywhere that says "it's this time". I'm assuming this is true for all languages, but we should look into it. If we want to support leaving the game, we'll need to verify that however we are getting the current time is difficult to tamper with, since terraria (and the mod) will be off during this time. We can definitely add some flag that says if the runner is entering a world without a new character (only 3 basic items/0:00 play time) and a new world (not sure how to verify this). I think we should skip this for the first version, but should follow it up pretty quickly. This is a good thing to put in the readme: future plans. It'll keep us on track throughout development and make sure we're working on the right features :)

Reply:
If it becomes annoying making it server sided will solve it all but we can cross that bridge when we get there

Reply:
for sure we need to split the project up into 1st version and possible future plans. ill do that in another pull request